datadog/resource_datadog_logs_pipeline_order_test.go

```diff
@@ -6,14 +6,14 @@ import (
 )
 
 const pipelinesConfig = `
-resource "datadog_logs_pipeline" "pipeline_1" {
+resource "datadog_logs_customer_pipeline" "pipeline_1" {
```

Comment:
These should all be `custom` right? 

```suggestion
resource "datadog_logs_custom_pipeline" "pipeline_1" {
```